Data/Sys/Shaders/FXAA Scanlines.glsl

```diff
@@ -0,0 +1,82 @@
+//			DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+//					Version 2, December 2004
+
+// Copyright (C) 2013 mudlord
+
+// Everyone is permitted to copy and distribute verbatim or modified
+// copies of this license document, and changing it is allowed as long
+// as the name is changed.
+
+//			DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+//	TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
+
+// 0. You just DO WHAT THE FUCK YOU WANT TO.

```

Comment:
I'm sure this is against the code style

Reply:
Amazingly, this seems to be a valid GPL license:

https://en.m.wikipedia.org/wiki/WTFPL
http://www.wtfpl.net

Reply:
WTFPL is a bad license. It's not widely recognized as something valid worldwide, it does not provide a warranty waiver, etc.

If you don't care at all about usage of what you release, I would suggest using either:
* The CC0 license: https://creativecommons.org/publicdomain/zero/1.0/
* The MIT license (which requires attribution -- but given that this ships as source it makes very little difference): https://opensource.org/licenses/MIT

FWIW I would strongly oppose adding WTFPL licensed components to Dolphin.

Reply:
I used the header already present in the fxaa file

Reply:
Couldn't you relicence it under MIT?

Reply:
> Couldn't you relicence it under MIT?

I don't know how to behave for the part dell'fxaa that was already present since 2013 by mudlord
I don't know if it is appropriate to remove it and see some other type of filtering

Reply:
it is OK?